### ISSUE TITLE
Webpack entry

### ISSUE THREAD

**Issue by dland512** [resir014/electron-react-starter#1]
Quick question for you, more out of curiosity than anything else. I noticed that you have the following defined in your webpack config file:

```
entry: {
    app: './client/index.js'
}
```

Wondering why you didn't use the `./init.js` as the webpack entry point? That way you can get babel transpiling and other webpack goodness with that file as well. Looks like many of the webpack/react starters do it the way you're doing it so I'm sure there's a valid reason, just wondering what it is.

Thanks for the starter!

**Comment by resir014** [comment]
Yeah, I didn't think of this through when I made this starter kit. I didn't know much Webpack back then, and I just want something that works right off the bat.

The original idea for it was to have the client and renderer process be compiled separately to for people who use IPC (inter-process communication) in their Electron apps. I know it could be handled better, but for now, this'll do.

When I got the time to get into this again, or find out a solution on how to handle this, I'll look into it again.

Thanks for your comments though!
